build/vsts-ci.yml

```diff
@@ -249,6 +246,14 @@ phases:
       msbuildVersion: 15.0
     continueOnError: true
 
+  - task: NuGetCommand@2 
```

Comment:
Do we need this? Aren't the regular NuGet feeds updated with every check-in?

Reply:
It publishes to ModelBuilder. I think we still need that so we won't mess up Machinelearning feed

Reply:
I don't understand. This is a PR into the `master` branch. You shouldn't be updating where the NuGet feeds are going in the `master` branch.

Reply:
Removed

## Resolve